CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing to goodpractice
+
+## Opening issues
+
+The easiest way to note any behavioural curiosities or to request any new
+features is by opening a [github
+issue](https://github.com/ropensci-review-tools/goodpractice/issues).
+
+
+## Development guidelines
+
+If you'd like to contribute changes to `goodpractice`, we use [the GitHub
```

Comment:
I'm in favour of one sentence per line but this is your package so you can ignore this :innocent: https://docs.ropensci.org/aeolus/